app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    old_file, new_file = command.split()[1], command.split()[2]
```

Comment:
` command, old_file, new_file = command.split()`

Read the checklist, you should add a check that `command` is correct
![image](https://user-images.githubusercontent.com/104218550/208380178-1313fe8a-33c1-43ed-b1c2-103b24de1db6.png)